backend/cli_commands/SyncCommand.py

```diff
@@ -172,3 +201,33 @@ def sync_folder():
     if not log_tracker.log_occurred:
         logging.info("Nothing was modified, no new metadata was saved")
     logger.removeHandler(log_tracker)
+
+
+def extract_topics_from_mcap(mcap_path: str) -> dict[str:dict]:
+    with storage.open(mcap_path, "rb") as f:
+        reader = make_reader(f)
+        schema_map = {
+            schema.id: schema.name for schema in reader.get_summary().schemas.values()
+        }
+        channel_message_counts = reader.get_summary().statistics.channel_message_counts
+        duration = get_duration_from_mcap(mcap_path)
+        topic_info = {}
+        for channel in reader.get_summary().channels.values():
+            topic = channel.topic
+            topic_type = schema_map.get(channel.schema_id, "Unknown")
+            message_count = channel_message_counts.get(channel.id, 0)
+            topic_info[topic] = {
+                "name": topic,
+                "type": topic_type,
+                "message_count": message_count,
+                "frequency": 0 if duration == 0 else message_count / duration,
```

Comment:
```suggestion
                 "frequency": 0 if duration == 0 else round(message_count / duration, 2),
```
One last thing I just noticed is that the frequency is not roundend down but shows all numbers after the decimalpoint
![grafik](https://github.com/user-attachments/assets/f4879708-c3cb-49c7-aaa4-4e298ef0a5a9)
I suggested a change to rund that down to 2 numbers after the decimal point